cantabular/dimensions.go

```diff
@@ -3,13 +3,16 @@ package cantabular
 import (
 	"context"
 
-	"github.com/pkg/errors"
-
+	"github.com/ONSdigital/dp-api-clients-go/v2/batch"
 	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular/gql"
 	dperrors "github.com/ONSdigital/dp-api-clients-go/v2/errors"
 	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/pkg/errors"
```

Comment:
Generally we keep 3rd part packages separate from ONS packages in the imports

Reply:
linter but done